tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": false,
+    "target": "es5",
+    "lib": ["es5", "es6", "es2016", "es2017.object", "dom"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "sourceMap": false,
+    "strictNullChecks": true
+  },
+  "files": ["tsdef/twilio-video-tests.ts"]
+}
```

Should have a new-line at the EOF.
```suggestion
}

```